run-tests.sh

```diff
@@ -13,6 +13,10 @@ set -o errexit
 # Quit on unbound symbols
 set -o nounset
 
+
```

Comment:
Leading whitespace line can be deleted; keep only one single empty line as separator.

Reply:
Removed 👍